addons/web/static/src/views/list/list_renderer.js

```diff
@@ -405,7 +405,10 @@ export class ListRenderer extends Component {
 
     focus(el) {
         el.focus();
-        if (["INPUT", "TEXTAREA"].includes(el.tagName)) {
+        if (
+            ["INPUT", "TEXTAREA"].includes(el.tagName) &&
+            ["text", "search", "URL", "tel", "password", "textarea"].includes(el.type)
+        ) {
             if (el.selectionStart === null) {
                 return;
             }
```

Comment:
Your fix makes the first nested `if (condition) {}` obsolete, isn't it ?

Also, this is not the first place in odoo where we need the same fix (e.g. look for useNumpadDecimal).
Maybe we should have an utility function i.e. `hasSelectionFeature(element)` ?

You don't have to do it in this PR, though (but you could as this is the master branch niark niark)